### DISCUSSION TITLE
Welcome to accweb Discussions!

### DISCUSSION THREAD

**Discussion by Kugelschieber** [assetto-corsa-web/accweb#135] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OtxiMOU** [comment]
Hello, I was anxiously waiting for version 2, I see that you have closed it, are you not going to continue with it?

**Reply by Kugelschieber** [reply]
Unfortunately, I don't have time to work on it. So I decided to keep v1 and support it until the game is no longer relevant.

**Comment by Warbolt** [comment]
is there any options to change the moderator role that he can change some settings? just start/stop server is kinda useless.
maybe change roles to:
- Admin
- GM (with options to change some settings)
- Moderator (start/stop server)

i'm missing the option that a moderator can e.g. change the track

**Reply by Kugelschieber** [reply]
If someone can edit the configuration, you can just make him an admin, because it doesn't matter if he can delete a server if he also could break the configuration.

**Comment by Schuedde** [comment]
Hi,
is there any manuell for installation and first steps with accweb. I am new in this and i installed it on my Ubuntu Vserver, to host a dedicated server. But if i start the ./accweb, he doesn t found a Server and on Webinterface he shows "no servers found". I dont know, what i am doing wrong. Maybe I did something wrong. It would be nice to find a step by step guide.
